kpi/filters.py

```diff
@@ -202,12 +203,17 @@ def filter_queryset(self, request, queryset, view):
             return queryset
 
         try:
-            q_obj = parse(q)
+            q_obj = parse(
+                q, default_field_lookups=ASSET_SEARCH_DEFAULT_FIELD_LOOKUPS
+            )
         except ParseError:
             return queryset.model.objects.none()
```

Comment:
As per discussed, I think it's better to handle the exceptions at the same place. 
Let's catch the `SearchQueryTooShortException` in the filter as well.

```python
except ParseError:
            return queryset.model.objects.none()
except SearchQueryTooShortException as e:
            raise e
```